docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -2,7 +2,9 @@
 import { SIDEBAR } from '../../config.ts';
 import { getLanguageFromURL } from '../../util.ts';
 const {currentPage} = Astro.props;
-const currentPageMatch = currentPage.slice(1);
+
+// Get the slug w/o a leading or trailing slash
+const currentPageMatch = currentPage.slice(1, (currentPage[currentPage.length-1] === '/') ? -1 : currentPage.length);
```

Comment:
we actually have a util helper for this!

```
import {addTrailingSlash} from '../../../util.ts';
```

If you're trying to do the opposite, you can add these helpers to `util.ts` and then use those, as well:

```

/** Remove \ and / from beginning of string */
export function removeLeadingSlash(path: string) {
  return path.replace(/^[/\\]+/, '');
}

/** Remove \ and / from end of string */
export function removeTrailingSlash(path: string) {
  return path.replace(/[/\\]+$/, '');
}
```

Reply:
Where is that imported from? I like the idea of the docs site being self-contained

Reply:
nvm, found it! https://github.com/snowpackjs/astro/blob/main/docs/src/util.ts

Reply:
Feel free to make a change like that, I probably won't touch this till tomorrow night.